examples/perf.js

```diff
@@ -19,16 +21,28 @@ const d = xys.map(([xu, yu]) =>
   )
 )
 
-console.log(`n = ${n}`)
-console.log(`d = [\n  ${d.map(d2 => `[${d2.join(', ')}]`).join(',\n  ')}\n]`)
-console.log()
+if (!quiet) {
+  console.log(`n = ${n}`)
+  console.log(`d = [\n  ${d.map(d2 => `[${d2.join(', ')}]`).join(',\n  ')}\n]`)
+  console.log()
+}
 
 console.time('HK/JS')
-console.log(getCycleJs(d))
+const cycleJs = getCycleJs(d);
 console.timeEnd('HK/JS')
+if (!quiet) {
+  console.log(cycleJs)
+}
 console.log()
 
 console.time('HK/WASM')
-console.log(await getCycleWasm(d))
+const cycleWasm = await getCycleWasm(d);
 console.timeEnd('HK/WASM')
+if (!quiet) {
+  console.log(cycleWasm)
+}
 console.log()
+
+if (JSON.stringify(cycleJs) !== JSON.stringify(cycleWasm)) {
+  console.log('RESULT MISMATCH!')
+}
```

Comment:
Please could you leave the performance testing script unmodified for now. But it looks like there would be some value in adding a unit test to the Mocha test suite which generates a 2D TSP problem and ensures that the JavaScript and WebAssembly implementations generate precisely identical results?

Reply:
No problem, undid these.